### PR TITLE
desktop terms page & privacy page & and the other feedbacks

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: components/PrivacyPage/PrivacyPage.tsx [hunk 1]
```diff
@@ -0,0 +1,1840 @@
...
+        )}
+        <div className="relative w-full">
+          <div
+            className="font-eigerdals 
+              text-[36px] md:text-[65px] 
+              pt-[10rem] pb-[40px] 
+              dark:text-[white] 
+              mx-4 md:mx-12"
+          >
+            Terms of Service
```
Comment (sweetmantech) [inline components/PrivacyPage/PrivacyPage.tsx]: this should say `Privacy Policy` right?
```suggestion
            Privacy Policy
```

**Review by SweetmanTech — APPROVED** [review]
one small comment

File: components/LandingPage/LandingContent.tsx [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 import WelcomeToCre8ors from "./sections/WelcomeToCre8ors"
 import InviteForCreator from "./sections/InviteForCreator"
 import Networking from "./sections/Networking"
 import Cre8orsWay from "./sections/Cre8orsWay"
 import Brands from "./sections/Brands"
 import OpenSoon from "./sections/OpenSoon"
 import { useTheme } from "../../providers/ThemeProvider"
-
-const SectionContainer = dynamic(() => import("./SectionContainer"), { ssr: false })
+import SectionContainer from "./SectionContainer"
```
Comment (sweetmantech) [inline components/LandingPage/LandingContent.tsx:9]: 💯

**Review by SweetmanTech — APPROVED** [review]
code looks good ✅

**Pull request closed without merging**